grape-swagger.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
   s.metadata['rubygems_mfa_required'] = 'true'
 
   s.required_ruby_version = '>= 2.7'
-  s.add_runtime_dependency 'grape', '~> 1.3'
+  s.add_runtime_dependency 'grape', '>= 1.3', '< 3.0'
```

Comment:
why?

Reply:
because grape is now version 2.0.0. `~> 1.3` explicitly means >= 1.3, < 2.0

Reply:
If you reference both these gems, you cannot update to grape 2.0 because of this constraint